### PR TITLE
Daily percent change

### PR DESCRIPTION
**Pull request by osvyg**
Resolves https://github.com/lazd/coronadatascraper/issues/19

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -207,4 +209,37 @@ async function generate(date, options = { findFeatures: true, findPopulations: t
...
+  return locations.map(location => {
+    let percentChange = "NA";
+    let locationDataYesterday = dataFromYesterday.find(d => d.county === location.county);
+    if (locationDataYesterday !== undefined) {
+      let casesDiff = location.cases - locationDataYesterday.cases;
+      if (casesDiff > 0) {
+        if (locationDataYesterday.cases > 0) {
+          percentChange = (casesDiff / locationDataYesterday.cases) * 100
+        } else {
+          percentChange = location.cases * 100
```
Comment (osvyg) [inline index.js]: I am not sure if this should be this or just 100

File: index.js [hunk 2]
```diff
@@ -207,4 +209,37 @@ async function generate(date, options = { findFeatures: true, findPopulations: t
...
+  let filePath = './' + path.join('dist', `data-${yesterday}.json`);
+  let dataFromYesterday = [];
+  try {
+    dataFromYesterday = await fs.readJSON(filePath);
+  } catch (err) {
+    console.error('  ❌ JSON file "%s" not found percentages will not be calculated: ', filePath, err);
+    return locations
+  }
+  return locations.map(location => {
+    let percentChange = "NA";
```
Comment (osvyg) [inline index.js]: should this just be 0?

**Comment by lazd** [comment]
So I'm actually working on doing this for the forthcoming timeseries data, where we know we have data from yesterday. Though your approach is valid, it assumes yesterday's data is present. I think rate change makes more sense in the context of timeseries.

Check out what I've done here: https://github.com/lazd/coronadatascraper/tree/timeseries

**Comment by osvyg** [comment]
Sounds good

**Pull request closed without merging**